pkg/reconciler/containersource/containersource_test.go

```diff
@@ -98,10 +96,10 @@ func TestAllCases(t *testing.T) {
 		}, {
 			Name: "error creating sink binding",
 			Objects: []runtime.Object{
-				NewContainerSourceV1Beta1(sourceName, testNS,
-					WithContainerSourceUIDV1B1(sourceUID),
-					WithContainerSourceSpecV1B1(makeContainerSourceSpec(sinkDest)),
-					WithContainerSourceObjectMetaGenerationV1B1(generation),
+				NewContainerSource(sourceName, testNS,
```

Comment:
I think typically we've kept the version attached to the functions here, ie `NewContainerSourceV1` and `WithContainerSourceUIDV1`

Reply:
ah nevermind, we seemed to have dropped that! resolving

Reply:
Yes, right now these functions are in [pkg/reconciler/testing/containersource.go](https://github.com/knative/eventing/blob/master/pkg/reconciler/testing/containersource.go) , where we have suffix to specify the versions, what I did was to put each version's functions into its own directory: `pkg/reconciler/testing/v1/containersource.go` and use the directory name to distinguish among the versions. 

I've seen other sources follow the same practice: [pkg/reconciler/testing/v1beta1/sinkbinding.go](https://github.com/knative/eventing/blob/master/pkg/reconciler/testing/v1beta1/sinkbinding.go)